tests/chart/test_auth_sidecar.py

```diff
@@ -93,15 +93,14 @@ def test_auth_sidecar_config_with_dag_server_enabled(self, kube_version):
             ],
         )
 
-        common_dagserver_sts_test_cases(docs, 3)
+        common_sts_test_cases(docs, 3, "release-name-dag-server", "StatefulSet")
```

Comment:
This applies to more than just `sts`
```suggestion
        common_pod_manager_test_cases(docs, 3, "release-name-dag-server", "StatefulSet")
```